### DISCUSSION TITLE
Issue, bluetooth connects to xbox controller fine but sends no output to gamecube

### DISCUSSION THREAD

**Discussion by obnoxiousmods** [darthcloud/BlueRetro#885] [Q&A]
### BlueRetro firmware version

 [v1.9.1 hw2 gamecube]

### BlueRetro firmware specification

HW2

### BlueRetro firmware variant

System specific

### BlueRetro hardware type

External adapter with detachable cord

### Manufacturer

Aliexpress Blueretro store

### System used

Nintendo GameCube

### Bluetooth controller brand & name

Xbox one controller

### What is problem? (only list ONE problem per report)

Bluetooth pairs properly but inputs arent sent to the gamecube

### What did you expect to happen?

Expect that inputs get sent to the gamecube

### Attach files like logs or Bluetooth traces here

Not sure what to do about this or how to diagnose..

**Comment by obnoxiousmods** [comment]
I can get it to connect to blueretro.io and lets me config it, have reflashed it a few times, tried universal and gamecube, same issue, can only think of it somehow being made wrong idk

**Comment by obnoxiousmods** [comment]
tried ps5 controller too, 100% pairs properly but gamecube gets nothing, tried every port in p1 and each by itself or reversed etc thing just wont work

**Comment by darthcloud** [comment]
I think your adapter is simply defective, that store is in no way affiliated to the BlueRetro project.

They are just asshole stealing the project name and giving nothing back.

**Comment by antsyclancy** [comment]
I have the same exact issue, I purchased the blue retro from the aliexpress page that is linked to on the official blue retro page. 


https://www.aliexpress.us/item/3256804476307142.html?gatewayAdapt=glo2usa
